### PR TITLE
Fix broken Pipeline tests

### PR DESCRIPTION
**Pull request by marty331**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -97,3 +97,9 @@ services:
     container_name: stripe
     ports:
       - "$STRIPE_MOCK_PORT:12112"
+
+  wiremock:
```
Comment (bowlofstew) [inline docker-compose.yml]: Can be nuked now, correct?

File: src/shared/cfg.py [hunk 2]
```diff
@@ -224,9 +224,7 @@ def DYNALITE_PORT(self):
 
     @property
     def SALESFORCE_BASKET_URI(self):
-        return self(
-            "SALESFORCE_BASKET_URI", "https://subhub.free.beeceptor.com?api-key="
-        )
+        return self("SALESFORCE_BASKET_URI", "http://127.0.0.1:3000/posts?api-key=")
```
Comment (bowlofstew) [inline src/shared/cfg.py]: Needed still?

File: src/shared/dynamodb.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 logger = get_logger()
 
 
-def random_label(length):
+def random_label(length: int) -> str:
```
Comment (bowlofstew) [inline src/shared/dynamodb.py]: nice improvement, courtesy of mypy.

File: docker-compose.yml (hunk 1 above)
Comment (marty331) [reply docker-compose.yml]: Yes, good catch, fixing now.

File: src/shared/cfg.py (hunk 2 above)
Comment (marty331) [reply src/shared/cfg.py]: Nope, will fix.

File: src/shared/dynamodb.py (hunk 3 above)
Comment (marty331) [reply src/shared/dynamodb.py]: yes, so thankful for mypy and it's courteous behavior...

**Pull request merged**